src/components/calendar/package.json

```diff
@@ -135,6 +135,10 @@
           {
             "prop": "labelHelp",
             "description": "Help text that appears at the bottom of the calendar grid"
+          },
+          {
+            "prop": "formatLong",
+            "description": "Format object for displayed text string"
```

Comment:
This should probably be expanded to say that it is options passed to `Intl.DateTimeFormat`